### DISCUSSION TITLE
Ground absorption

### DISCUSSION THREAD

**Discussion by IsotoCedex** [Universite-Gustave-Eiffel/NoiseModelling#612] [Q&A]
Good morning,
If I'm not mistaken, by default, the ground absorption in NM is "0".
Where can I change it, so that it is "1"?
Thank you!

**Comment by nicolas-f** [comment]
Hi,

If you mean by using a parameter/settings, there is none.

If you ask about the source code, gPath are computed here:

https://github.com/Universite-Gustave-Eiffel/NoiseModelling/blob/4.X/noisemodelling-pathfinder/src/main/java/org/noise_planet/noisemodelling/pathfinder/ProfileBuilder.java#L1821

It would be easier to define soft ground of your scene than gathering all roads parking and all hard ground locations.

**Comment by IsotoCedex** [comment]
I have tried both options. For the model I am calculating, it is not feasible to introduce a layer with polygons with alpha=1, or with alpha=0. It would be a very heavy layer.
![image](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/assets/87642224/739319c1-c4cc-4a40-85a6-f3806bb2ff90)

Even if you enter it for the entire domain, the calculation is greatly slowed down.
For that reason, I would like to set alpha=1 by default in the code.
In which line can I change from alpha=0 to alpha=1 by default?

**Reply by nicolas-f** [reply]
You need to explode/split and simplify these geometries before supplying them to NoiseModelling. 

For each square calculation sub-domains NoiseModelling will query your ground table, so the geometry **should not** be unified and should be indexed.

The algorithm has to be rewritten for != 0 so it is not a simple change in source code.

**Comment by IsotoCedex** [comment]
You mean I can't use one polygon for the entire domain?

**Reply by nicolas-f** [reply]
You can but it is not efficient. As NoiseModelling is splitting your domain according to the maximal propagation distance into smaller areas, when it is gathering data for each small area it will query your database using the small envelope, but your layer groundType contain complex large polygons.

In order to reduce the computation time, you need to split into smaller polygons before supplying them to NoiseModelling.

https://maartenvandenhoven.com/blog/turning-a-polygon-into-a-grid-with-qgis/

**Comment by IsotoCedex** [comment]
Thank you so much. I understand now!!
